server/app/filters/CiviFormProfileFilter.java

```diff
@@ -43,6 +43,8 @@ public CiviFormProfileFilter(Materializer mat, ProfileUtils profileUtils) {
   private boolean shouldRedirect(Http.RequestHeader requestHeader) {
     return NonUserRoutes.noneMatch(requestHeader)
         && OptionalProfileRoutes.noneMatch(requestHeader)
+        // Ensure auth0 can call this endpoint without redirecting
```

Comment:
```suggestion
        // Ensure authorization server (IDP) can call this endpoint without redirecting
```

This doesn't seem specific to auth0